orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/aws/cloudformation/WaitForCloudFormationCompletionTask.java

```diff
@@ -72,8 +72,15 @@ public TaskResult execute(@Nonnull Stage stage) {
           isChangeSet
               ? getChangeSetInfo(stack, stage.getContext(), "status")
               : getStackInfo(stack, "stackStatus");
-      if (isComplete(status) || isEmptyChangeSet(stage, stack)) {
+      if (isComplete(status)) {
         return TaskResult.builder(ExecutionStatus.SUCCEEDED).outputs(stack).build();
+      } else if (isEmptyChangeSet(stage, stack)) {
+        String changeSetName = (String) result.get("changeSetName");
+        log.debug("CloudFormation ChangeSet {} empty. Requesting to be deleted.", changeSetName);
```

Comment:
`debug` or `info`? again, just calling it out to make sure it's appropriate level - I think most installation have `debug` logging turned off

Reply:
I'll change this one to info as it is taking an action 👍